patternfly/patternfly-elements
# fix(create-element): missing docs on create

## <Pull request description>




### Related issues

- (#1908) Docs folder and index.md missing on create-element


### What has changed and why



- Added docs folder and basic `index.md` file on initial creation of a new element


### Testing instructions



1. `npm ci`
2. `npm run build -w @patternfly/create-element`
3. `npm run new`
4. complete create element wizard
5. `elements/{{new-element}}/docs` folder should exist
6. `elements/{{new-element}}/docs/index.md` file should exist

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
